dsl/pace/dsl/dace/orchestrate.py

```diff
@@ -261,14 +330,10 @@ def __sdfg__(self, *args, **kwargs):
                     simplify=False,
                 )
             else:
-                if os.path.isfile(self.lazy_method._load_sdfg):
-                    self.daceprog.load_sdfg(
-                        self.lazy_method._load_sdfg, *args, **kwargs
-                    )
+                if os.path.isfile(sdfg_path):
+                    self.daceprog.load_sdfg(sdfg_path, *args, **kwargs)
```

Comment:
Would it recompile if the sdfg_path is incomplete/broken? Just wondering if the workflow is to delete the sdfg cache or would it pick up where it left?